denizen_scripts/survival/repo_link/shops/market_system.dsc

```diff
@@ -52,13 +52,14 @@ market_system_data:
       lore: <&a>Trade Goods!
   items:
     # Crafting
-    netherite_ingot:
-      category: crafting
-      minimum_value: 2200
+#    -- Nether stuff can come later, this is currently OP.
+#    netherite_ingot:
+#      category: crafting
+#      minimum_value: 2200
```

Comment:
This isn't "removing" content, it's replacing it with bloat. If the intent here was to note that this was what previously existed, we have [`[git blame]`](https://github.com/Adriftus-Studios/network-script-data/blame/master/denizen_scripts/survival/repo_link/shops/market_system.dsc) which would outline the history of this file.
```suggestion
```